web/app/js/components/Tap.jsx

```diff
@@ -276,6 +291,7 @@ class Tap extends React.Component {
   }
 
   updateQuery = query => {
+    this.resetTapResults();
```

Comment:
For consistency with the top page, how would you feel about moving this `resetTapResults` call to the beginning of the `startTapStreaming` method instead? That way the results only clear once you start a new tap, but not when you change the filters. That would match how the results are cleared on the top page.